net-dns/dnsdist/files/1.9.8-fix-compat-with-boost-lockfree-1.87.0.patch

```diff
@@ -0,0 +1,90 @@
+Patch from: https://github.com/PowerDNS/pdns/pull/15181
+Fixed file paths for dnsdist tarball.
+
+Signed-off-by: Holger Hoffstätte <holger@applied-asynchrony.com>
+
+From: Remi Gacogne <remi.gacogne@powerdns.com>
+Date: Mon, 10 Feb 2025 11:24:28 +0100
+Subject: [PATCH] dnsdist-1.9.x: Fix compatibility with boost::lockfree >= 1.87.0
```

Comment:
Just to double check: this is backwards compat w/ <1.87?

Reply:
I honestly do not know :sob: 
Let me try to rebuild with current stable boost-1.85.0-r1 and see what happens.

Reply:
It compiles fine against 1.85.0-r1, so I guess that settles that.
Honestly after looking at that change in upstream I cannot tell whether this was broken before or what _should_ happen in the first place, from a concurrency vs. compiler perspective. :exploding_head: